src/SFA.DAS.AssessorService.Database/PostDeploymentScripts/EPAO_Migration_Part1_Certificates.sql

```diff
@@ -0,0 +1,75 @@
+﻿-- migration for EPAO service
+WITH standardversions 
```

Comment:
Wouldn't mind a chat through this script if possible

Reply:
Anytime